.vsconfig

```diff
@@ -1,53 +1,55 @@
 {
   "version": "1.0",
   "components": [
-    "Microsoft.VisualStudio.Component.Roslyn.Compiler",
-    "Microsoft.VisualStudio.Component.Roslyn.LanguageServices",
+    "Component.CPython39.x64",
+    "Microsoft.Component.CodeAnalysis.SDK",
```

Comment:
How big is this SDK? Not that many people touch this and CLI builds do this without any install. If it's not too big, then I'm all for it.

Reply:
According to the installer UI, adding the .NET Compiler Platform SDK requires 1 MB.